### PR TITLE
8696kuryg Demo Typescript app with linting

### PR DESCRIPTION
**Pull request by matcor-bodc**

### CODE REVIEW

File: typescript-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
+
+## Getting Started
+
```
Comment (thogar-computer) [inline typescript-demo/README.md:4]: I know this is the default readme, can we add in the npm install step and then the npm lint commend as well to show a user how to lint?

**Review by thogar-computer — COMMENTED** [review]
Happy with liniting and seeing it working and failing in pipeline

File: typescript-demo/README.md (hunk 1 above)
Comment (matcor-bodc) [reply typescript-demo/README.md:4]: I had realised the default readme didn't include that. Pretty odd since the lint comes with it. I'll add that in

Comment (matcor-bodc) [reply typescript-demo/README.md:4]: done

**Review by thogar-computer — APPROVED** [review]

**Pull request merged**